### PR TITLE
extend section 4.3 and add results there

### PR DESCRIPTION
**Pull request by mikhail-vlasenko**

### CODE REVIEW

File: demos/original_vs_updated_table.ipynb
Comment (iasonsky) [inline demos/original_vs_updated_table.ipynb:1]: are the changes here necessary?

Comment (mikhail-vlasenko) [reply demos/original_vs_updated_table.ipynb:1]: they are as minimal as i can make them. i only added code and didn't commit the new meta info and outputs.

the changes make it possible to get the results for the euivariant runs

Comment (iasonsky) [reply demos/original_vs_updated_table.ipynb:1]: oh you added an extra cell right? I was seeing the changes from github and didn't realise

Comment (mikhail-vlasenko) [reply demos/original_vs_updated_table.ipynb:1]: 1 cell and 1 line yes

File: Blogpost.md [hunk 1]
```diff
@@ -195,7 +196,7 @@ points (2.28%) is less pronounced but still noticable in case of full finetuning
 |    | Method        | Architecture-Transformation        |   Prefinetune Top1 Acc |   Finetune Top1 Acc |
```
Comment (iasonsky) [inline Blogpost.md:196]: We will need to update the whole table once all runs are complete

**Review by iasonsky — APPROVED** [review]
lgtm

**Pull request merged**